riscv-page-tables/src/page_table.rs

```diff
@@ -554,6 +567,35 @@ impl<T: PagingMode> PageTableInner<T> {
         }
     }
 
+    /// Updates the `vaddr` mapping to not point to `paddr` and returns old SupervisorPageAddr.
```

Comment:
is there an extra 'not' in this comment? Calling update_page_addr(paddr) should cause vaddr to point to paddr, or am I lost?